app/controllers/theme_controller.rb

```diff
@@ -0,0 +1,3 @@
+class ThemeController < ApplicationController
```

Comment:
Can we please rename this to `ThemesController` instead of using it as singular?

Reply:
Thanks @Levia I will get that sorted right now!